controller/tap/server.go

```diff
@@ -37,6 +37,7 @@ type GRPCTapServer struct {
 	k8sAPI              *k8s.API
 	controllerNamespace string
 	clusterDomain       string
```

Comment:
is clusterDomain actually used for anything?  does it need to be part of this struct?

Reply:
Good catch :+1: I'll remove that.